File: src/llm/fallback.ts
import { FallbackAdvice } from '../types/index.js';

export const FALLBACK_ADVICE: FallbackAdvice = {
  advice: 'Sit up straight, pull shoulders back, tuck chin slightly, and raise monitor to eye level.',
  confidence_score: 60,
};

export function getFallbackAdvice(): FallbackAdvice {
  return FALLBACK_ADVICE;
}

File: backend/archives/internal/llm/fallback.go
package llm

import "posture-backend/internal/models"

func GetFallbackAdvice() models.PostureAdvice {
	return models.PostureAdvice{
		Advice:          "Sit up straight, pull your shoulders back, and adjust your monitor to eye level. Take a 30-second stretch break.",
		ConfidenceScore: 60,
	}
}
